docs/tutorials/ipfs-browser-devenv/example/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+  <head><title>Page title</title></head>
```

Comment:
```suggestion
  <head><title>Editor Example</title></head>
```

Should defs have a title